numpyro/distributions/transforms.py

```diff
@@ -106,6 +106,16 @@ def inverse_shape(self, shape):
         """
         return shape
 
+    # Allow for pickle serialization of transforms.
```

Comment:
This follows [Pyro](https://github.com/pyro-ppl/pyro/blob/43dbb2e67331bd3aa6c73bb5483825432a8b7145/pyro/distributions/torch_patch.py#L39).